### PR TITLE
removed code to resolve signal abort issue

### PR DESCRIPTION
**Pull request by gohkokboon**

### CODE REVIEW

**Comment by lauleehong** [comment]
awaiting decada cloud to be online, for testing

File: src/CryptoEngine/crypto_engine.cpp [hunk 1]
```diff
@@ -245,6 +245,8 @@ std::string CAPEMFormatter(std::string s)
  */
 bool X509IssuerInfo(char* buf, size_t size, const mbedtls_x509_crt* crt)
 {
+    printf("\r\n1\r\n");
```
Comment (lauleehong) [inline src/CryptoEngine/crypto_engine.cpp]: remove

Comment (lauleehong) [reply src/CryptoEngine/crypto_engine.cpp]: Also, line306 need formatting

**Comment by lauleehong** [comment]
I think we can merge this, since the fix has been verified to solve the bug. @gohkokboon ok?

File: src/CryptoEngine/crypto_engine.cpp [hunk 2]
```diff
@@ -288,15 +284,16 @@ bool X509CADecoder(std::string ssl_ca, ssl_ca_params& ca_params)
...
     strcpy(ssl_ca_buf, ssl_ca.c_str());
     
     const uint32_t buf_size = 512;
     char *buf = new char[buf_size];
     
     mbedtls_x509_crt x509_root_ca;
     mbedtls_x509_crt_init(&x509_root_ca);
-    mbedtls_x509_crt_parse(&x509_root_ca, (const unsigned char*) ssl_ca_buf, sizeof(ssl_ca_buf));
-    
+    int res = mbedtls_x509_crt_parse(&x509_root_ca, (const unsigned char*) ssl_ca_buf, sizeof(ssl_ca_buf));
```
Comment (lauleehong) [inline src/CryptoEngine/crypto_engine.cpp]: `res` is not used?

Comment (gohkokboon) [reply src/CryptoEngine/crypto_engine.cpp]: removed

**Comment by gohkokboon** [comment]
> I think we can merge this, since the fix has been verified to solve the bug. @gohkokboon ok?

**Pull request merged**